pyanaconda/ui/gui/spokes/software.py

```diff
@@ -396,7 +396,7 @@ def refresh(self):
             elif firstEnvironment:  # manual installation
                 # for manual installs that don't have a default provided by the install class
                 # just tick the first radio button and select the first environment
-                if not self.environment_valid:
+                if not self.environment_valid or self.environment is None:
```

Comment:
Thanks for the explanation. I think a comment right before the condition explaining what ``self.environment`` being ``None`` means would do.